Donsol/ContentView.swift

```diff
@@ -31,14 +20,45 @@ struct ContentView: View {
 
                 HeaderView(game: game)
                 
-                
                 HStack {
-                    CardButtonView(game: game, isFlipped: $topRightCard_isFlipped, isEnabled: $isEnabled, card: game.deck.drawCard())
-                    CardButtonView(game: game, isFlipped: $topLeftCard_isFlipped, isEnabled: $isEnabled, card: game.deck.cards[0])
+                    if game.room.count > 0 {
```

Comment:
It would be cleaner to use a `Grid` and use a `ForEach` block to render the contents of `game.room`